src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -424,15 +426,29 @@
 				.examples("on flight toggle:",
 						"	if {game::%player%::playing} exists:",
 						"		cancel event")
-				.since("2.2-dev36");
+				.since("2.2-dev36"); 
 		if (Skript.classExists("org.bukkit.event.player.PlayerLocaleChangeEvent")) {
 			Skript.registerEvent("Language Change", SimpleEvent.class, PlayerLocaleChangeEvent.class, "[player] (language|locale) chang(e|ing)", "[player] chang(e|ing) (language|locale)")
 					.description("Called after a player changed their language in the game settings. You can use the <a href='expressions.html#ExprLanguage'>language</a> expression to get the current language of the player.",
-						"This event requires Minecraft 1.12+.")
+							"This event requires Minecraft 1.12+.")
```

Comment:
all the doc changes above are completely unrelated to this pr

Reply:
why not

Reply:
@Blueyescat this should at least be in a separate commit

Reply:
Whyy lol it just adds a missing indention

Reply:
I was talking about the doc link fixes - they should be either in a separate ~~PR~~ commit, or noted in the commit description. But that's just a good practice for you to follow in the next PR, shouldn't be too problematic this time if it only happened once